projects/turtle/turtle-project.html

```diff
@@ -121,13 +121,11 @@ <h3 id="types-linting-documentation-10-points-total">6. Types, Linting, Document
 <h3 id="above-and-beyond-breaks-up-complex-functions-5-points-total">7. Above and beyond – breaks up complex functions (5 points total):</h3>
 <p>Avoid any single function from becoming <em>too</em> complex. When programming in general, if you find yourself writing a very complex function, it’s likely you should break it down further into simpler functions. To earn credit for this, break your most complex component procedure besides <code>main</code> down further into simpler, but still meaningfully named procedures. To earn credit for this, your component procedure should <em>call</em> these simpler procedures.</p>
 <h3 id="above-and-beyond-try-something-fun-5-points-total">8. Above and beyond – try something fun! (5 points total):</h3>
-<ol start="11" type="1">
-<li>Impress us by trying out something you find interesting! Describe what you are attempting here in the file’s <code>docstring</code> so we know what to look for. Remember, a <code>docstring</code> can span more than one line, it just ends with the <code>"""</code>. Some ideas you could try for here:</li>
-</ol>
```

Comment:
Removed the ordered list here—felt weird for a paragraph to suddenly start at "11." Unless I'm missing something.